dir.props

```diff
@@ -125,7 +125,9 @@
 
   <!-- Create a collection of all project.json files for dependency updates. -->
   <ItemGroup>
-    <ProjectJsonFiles Include="$(SourceDir)**/project.json" />
+    <!-- Exclude PackageFiles tool-runtime and test-runtime project.json files. -->
+    <ProjectJsonFiles Include="$(SourceDir)**/project.json"
+                      Exclude="$(SourceDir)Microsoft.DotNet.Build.Tasks\PackageFiles\**\project.json" />
```

Comment:
What I've been seeing from my tests is that any dir separators generated as part of `**` are being created as `\` by MSBuild, and the `/` in `/project.json` isn't preserved. (Maybe `**/` is conceptually what's being picked up.) It ends up with paths like `E:\git\buildtools\src/Microsoft.DotNet.Build.Tasks.Packaging\src\project.json`.

Making both use `\` for consistency. `ProjectJsonFiles` isn't currently used for any `Exec` calls, so even if buildtools were xplat I think this would be safe.